content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -167,13 +194,29 @@ and installing the sidecar injector webhook configuration on the remote cluster
         pilot:
           configMap: true
         istiodRemote:
-          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/:ENV:cluster=${REMOTE_CLUSTER_NAME}:ENV:net=network1
+          injectionURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/inject/cluster/${REMOTE_CLUSTER_NAME}/net/network1
         base:
           validationURL: https://${EXTERNAL_ISTIOD_ADDR}:15017/validate
     EOF
     {{< /text >}}
 
-    Then, install the configuration on the remote cluster:
+1.  If you are using an IP address for the `EXTERNAL_ISTIOD_ADDR`, instead of a proper DNS hostname,
```

Comment:
Does it make sense to put the `If you are using an IP address...` blocks inside info boxes similar to the first one? Or does that cause formatting issues?

Reply:
Or maybe it's the lack of placing the warning box in the following blocks?

Reply:
Yes, that was my first attempt, but unfortunately there's a long known hugo problem with nested text segments inside of other tips, warnings, or tabs. It loses some of the indentation, but the sed commands I'm adding have very specific indentation format for insert/append lines.